simapp/Makefile

```diff
@@ -295,11 +295,7 @@ setup-testnet-keys:
 	-`echo "decorate bright ozone fork gallery riot bus exhaust worth way bone indoor calm squirrel merry zero scheme cotton until shop any excess stage laundry" | wasmd keys add acc0 --recover`
 	-`echo "wealth flavor believe regret funny network recall kiss grape useless pepper cram hint member few certain unveil rather brick bargain curious require crowd raise" | wasmd keys add acc1 --recover`
 
-# default testnet is with IBC
 testnet: setup-testnet
-	spawn local-ic start ibc-testnet
-
-testnet-basic: setup-testnet
 	spawn local-ic start testnet
```

Comment:
no longer doping a basic, basic is the IBC or ICS based testnet only. sh-testnet for basic